src/components/common/AppMenu.vue

```diff
@@ -3,7 +3,14 @@
     <div v-if="session.signedIn" class="user-box">
       <div class="user-box-address">
         <div>
-          <h3>Your Address</h3>
+          <h3
+            v-if="
+              session.addressRole && session.addressRole !== `stash/controller`
```

Comment:
Is this accounting for non-Polkadot addresses? I mean, do non-Polkadot addresses have `stash-controller` as their accountRole?

Reply:
I think, by looking at session.js, that it would be `undefined` on those cases